modules/api/static/schema.yml

```diff
@@ -1823,7 +1823,9 @@ components:
       type: object
       properties:
         Candidates:
-          $ref: '#/components/schemas/Candidate'
+          type: array
```

Comment:
Can you apply this to modules/api/static/schema-v0.0.4-dev.yml as well? please

Reply:
hey @xlecours thanks for the review! i added other fixes here to the v0.0.4-dev schema as well